homeassistant/components/ecobee/climate.py

```diff
@@ -529,14 +581,33 @@ def hvac_action(self):
     def extra_state_attributes(self):
         """Return device specific state attributes."""
         status = self.thermostat["equipmentStatus"]
-        return {
+        attributes = {
             "fan": self.fan,
             "climate_mode": self._preset_modes[
                 self.thermostat["program"]["currentClimateRef"]
             ],
             "equipment_running": status,
             "fan_min_on_time": self.thermostat["settings"]["fanMinOnTime"],
+            "ventilator_type": self.thermostat["settings"]["ventilatorType"],
         }
+        if self.thermostat["settings"]["ventilatorType"] != "none":
+            attributes["vent"] = self.thermostat["settings"]["vent"]
+            attributes["ventilator_min_on_time"] = self.thermostat["settings"][
+                "ventilatorMinOnTime"
+            ]
+            attributes["ventilator_min_on_time_home"] = self.thermostat["settings"][
+                "ventilatorMinOnTimeHome"
+            ]
+            attributes["ventilator_min_on_time_away"] = self.thermostat["settings"][
+                "ventilatorMinOnTimeAway"
+            ]
+            attributes["is_ventilator_timer_on"] = self.thermostat["settings"][
+                "isVentilatorTimerOn"
+            ]
+            attributes["ventilator_off_date_time"] = self.thermostat["settings"][
+                "ventilatorOffDateTime"
+            ]
```

Comment:
Why do we need all those attributes? Should they have been separate entities? Are they really needed?